### PR TITLE
Add searchbar to Stories page

### PR DESCRIPTION
**Pull request by tnwoodard**
This pull request addresses issue #507. The search bar was built using HTML, Bulma CSS and Javascript. The search bar uses your input search the DOM for any matching content in each card.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,14 +1,14 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    activesupport (6.0.5)
```
Comment (bebatut) [inline Gemfile.lock:4]: Could you revert your changes on this file? Thanks

File: _includes/youtube-embed.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<div class="youtube">
```
Comment (bebatut) [inline _includes/youtube-embed.html:1]: It feels like these changes might not belong to this PR

File: _layouts/posts.html [hunk 3]
```diff
@@ -38,4 +46,5 @@
         </div>
     </div>
 {% endfor %}
-</div>
\ No newline at end of file
+</div>
+<script src="/js/posts.js"></script>
```
Comment (bebatut) [inline _layouts/posts.html:50]: It might be good to put the script directly in the `_layouts/base.html` with the other js scripts (with a if checking if name of the page is post)

**Review by bebatut — COMMENTED** [review]
Thanks @tnwoodard. It looks good :)

File: Gemfile.lock (hunk 1 above)
Comment (tnwoodard) [reply Gemfile.lock:4]: Yes, sorry about that.

File: _includes/youtube-embed.html (hunk 2 above)
Comment (tnwoodard) [reply _includes/youtube-embed.html:1]: Yes, I will get rid of it.

File: _layouts/posts.html (hunk 3 above)
Comment (tnwoodard) [reply _layouts/posts.html:50]: Ok, I will add the js script with the others.

**Comment by tnwoodard** [comment]
Thank you for the feedback @bebatut. I was able to address it.

**Comment by bebatut** [comment]
It looks good. Thanks a lot for the work here
